tests/FSharp.Data.GraphQL.Tests/AstValidationTests.fs

```diff
@@ -11,8 +11,8 @@ open FSharp.Data.GraphQL.Types
 
 module internal GQLProblemDetails =
 
-    let CreateValidation message = GQLProblemDetails.CreateWithKind (message, Validation)
-    let CreateValidationFor path message = GQLProblemDetails.CreateWithKind (message, Validation, path)
+    let CreateValidation message = GQLProblemDetails.CreateWithKind (message, None, Validation)
+    let CreateValidationFor path message = GQLProblemDetails.CreateWithKind (message, None, Validation, path)
```

Comment:
And with optionals we can place ex at the end and not specify here

Reply:
I thought about that possibility too (i.e.: just append the exception parameter to the list of parameters). But the reason why I put the exception argument in the middle is because the message and the exception are tightly related, almost as if they were two members of the same type. So it makes more sense to have them one next to the other than separated by other parameters.

Reply:
Not necessary anymore with the now extra `IGQLExceptionError` interface.